code/controllers/master_controller.dm

```diff
@@ -71,6 +71,12 @@ datum/controller/game_controller/proc/setup_objects()
 			T.broadcast_status()
 
 
+	//Spawn the contents of the cargo warehouse
+	admin_notice("<span class='danger'>Populating cargo bay..</span>", R_DEBUG)

```

Comment:
ill reduce it down to one line, the spawn cargo stock already outputs a line about the time to completion